src/utils/useDataApiWithFeedback.ts

```diff
@@ -26,7 +28,12 @@ function useDataApiWithFeedback() {
 
             if (result && isMutationResult(result)) {
               if (result.rejection) {
-                const { reason } = result.rejection as Rejection;
+                let { reason } = result.rejection as Rejection;
+                if (reason === 'EXTERNAL_TOKEN_INVALID') {
+                  handleLogout();
+                  reason =
+                    'You have been logged out redirecting to external homepage';
```

Comment:
I wonder if this might confuse some users. Would it be clearer to say that their session has expired and they need to login again?

Reply:
Cool, I can update it to be clearer is `Your session has expired, you will need to log in again through the external homepage.` Although this message, depending on how fast the redirect is, might not be displayed for very long.

Reply:
That seems good. If it doesn't stay up that long and it looks like too much to read in time, we could probably cut the `through the external homepage` part